npm-shrinkwrap.json

```diff
@@ -1,322 +0,0 @@
-{
```

Comment:
I see. According to more recent `npm` documentation, `devDependencies` are not included in a shrinkwrap:


> Since npm shrinkwrap is intended to lock down your dependencies for production use, devDependencies will not be included unless you explicitly set the --dev flag when you run npm shrinkwrap.

Rather than remove the shrinkwrap, do you think it's preferable to re-generate it using a more current version of `npm`, which would presumably exclude `devDependencies` as desired? In my mind, this then serves the goal of locking down the production dependency chain, while also excluding the `devDependencies`.